### PR TITLE
MULE-19584: Add Source Lifecycle logging

### PR DESCRIPTION
**Pull request by ndinu**

### CODE REVIEW

**Comment by ndinu** [comment]
@tanumutreja  please take a look when you have a moment.

File: modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java [hunk 1]
```diff
@@ -629,22 +629,22 @@ protected void doInitialise() throws InitialisationException {
         boolean isPrimaryPollingInstance = clusterService.isPrimaryPollingInstance();
         if (primaryNodeOnly) {
           LOGGER
-              .debug("Message source '{}' on flow '{}' is being initialised, it is running on the primary node and it must only run on this node.",
+              .debug("Message source '{}' on flow '{}' running on the primary node is initializing. Note that this Message source must run on the primary node only.",
                      sourceModel.getName(), getLocation().getRootContainerName());
         } else {
           LOGGER
-              .debug("Message source '{}' on flow '{}' is being initialised, there are no cluster restrictions regarding which node must run the source. This node {} the primary node.",
+              .debug("Message source '{}' on flow '{}' is initializing. This {} the primary node of the cluster.",
```
Comment (tanumutreja) [inline modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java:636]: Minor typo. Please add "is" i.e.  "This {} is the primary node of the cluster."

Comment (ndinu) [reply modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java:636]: The {} placeholder will be transformed into either "is" or "is not" depending if it is the primary node or not.

**Review by tanumutreja — APPROVED** [review]
Approved - log messages.

**Pull request merged**